layouts/podcast/summary.html

```diff
@@ -0,0 +1,9 @@
+<div class="summary">
+  <div class="titles">
+    <a href="{{ .RelPermalink }}">
+      <h2 class="title">Ep.{{.Params.episode}}: {{ .Title }}</h2>
```

Comment:
Same as above: `EP{{.Params.episode}}: {{.Title}}`

Reply:
Fixed in de62bf3!